dfusion_rust_core/src/models/account_state.rs

```diff
@@ -35,7 +35,7 @@ impl AccountState {
                 .enumerate()
                 .map(|(account, token_balances)| {
                     (
-                        H160::from(account as u64), // TODO - these are not accurate addresses.
+                        H160::from_low_u64_be(account as u64), // TODO - these are not accurate addresses.
```

Comment:
to bad they changed 😞

Reply:
Yes................... Although it makes a lot of sense. There shouldn't be a `From` conversion from integers to hash - it doesn't really make sense, especially since it was endianness dependent.

Reply:
Is this todo still necessary or was this just a point of confusion a long time ago? I know that snapp uses account ids, while stableX uses addresses.

Reply:
I think it's less a TODO and more a comment that the account IDs are not real addresses.